lms/templates/ux/reference/teams-create.html

```diff
@@ -0,0 +1,200 @@
+<%block name="nav_skip">${"#seq_content" if section_title else "#course-content"}</%block>
+</%block>
+
+<%block name="js_extra">
+
+  <script type="text/javascript">
+    $(function(){
+        $(".ui-accordion-header a, .ui-accordion-content .subtitle").each(function() {
+          var elemText = $(this).text().replace(/^\s+|\s+$/g,''); // Strip leading and trailing whitespace
+          var wordArray = elemText.split(" ");
+          var finalTitle = "";
+          if (wordArray.length > 0) {
+            for (i=0;i<=wordArray.length-1;i++) {
+              finalTitle += wordArray[i];
+              if (i == (wordArray.length-2)) {
+                finalTitle += "&nbsp;";
+              } else if (i == (wordArray.length-1)) {
+                // Do nothing
+              } else {
+                finalTitle += " ";
+              }
+            }
+          }
+          $(this).html(finalTitle);
+        });
+      });
+  </script>
+
+</%block>
+
+<div class="container">
+  <div class="course-wrapper">
+    <section class="course-content" id="course-content">
+
+    <header class="page-header">
+      <div class="header-main">
+        <nav class="breadcrumbs">
+          <a class="nav-item" href="">Solar Energy</a> <span class="icon fa-angle-right" aria-hidden="true"></span>
+        </nav>
+        <h2 class="page-title">Create a New Team</h2>
+        <p class="page-description">If you cannot find an existing team to join or would like to team up with a group of friends, create a new team.</p>
+      </div>
+    </header>
+
+    <div class="page-content">
+
+      <form class="create-team">
+      <fieldset class="required-wrapper">
+          <legend>Required Information</legend>
+
+          <div class="form-field required">
+            <label for="team-name" class="label">Team Name (Required)</label>
+            <input id="team-name" aria-required="true" class="form-input" />
+            <span class="tip tip-input">The name that will identify your team</span>
+          </div>
+
+          <div class="form-field required has-errors">
+            <label for="team-description" class="label">Team Description (Required)</label>
+            <textarea id="team-description" aria-required="true" class="form-input"></textarea>
+            <span class="tip tip-input">A short description of the team to help other students understand the goals or directives the team is pursuing</span>
+          </div>
+        </fieldset>
+        <fieldset class="optional-wrapper">
+          <legend>Optional Characteristics</legend>
+          <p class="description">Help other students find and join your team by specifying characteristics. The more limitations you add, the fewer students may be interested in joining, so choose carefully.</p>
+          <div class="form-field">
+            <label for="team-language" class="label">Language</label>
+            <select id="team-language" name="team-language" class="form-select">
```

Comment:
Ahhh, now I see what you did with input naming. Hmmm, you can disregard [my feedback above](https://github.com/edx/edx-platform/pull/7859/files#r29619870) if we're cool with this convention and can accommodate the following with it:

* text input
* large text input
* radio buttons
* checkboxes
* select fields
* email/telephone/contextual text inputs
* ranges